redbeat/schedulers.py

```diff
@@ -372,8 +372,10 @@ def maybe_due(self, entry, **kwargs):
 
     def tick(self, min=min, **kwargs):
         if self.lock:
-            logger.debug('beat: Extending lock...')
-            redis(self.app).pexpire(self.lock_key, int(self.lock_timeout * 1000))
+            logger.debug('beat: Extending lock by {} seconds...'.format(self.lock_timeout))
+            res = redis(self.app).pexpire(self.lock_key, int(self.lock_timeout * 1000))
+            if res != 1:
+                logger.warning('The key does not exist')
```

Comment:
I'm not sure why this message is needed. Could you either remove this or make it more obvious what action a user should take if they see this message.

Reply:
See https://github.com/sibson/redbeat/issues/83

Redbeat happily keeps working, even though it no longer owns the lock.

Reply:
If we've lost the lock would it be better to do some type of shutdown/exit rather than log an error that is easily missed or ignored?